### ISSUE TITLE
[BUG] [Server] 2.77 Illegal furnace

### ISSUE THREAD

**Issue by unicornbloods** [CalaMariGold/Rebirth-Of-The-Night#225]
ISSUES NOT USING OUR TEMPLATE WILL NOT BE CONSIDERED AND WILL BE CLOSED!

**Describe the bug**

I have an illegal furnace recipe error when trying to join my server

**To Reproduce**

join my server

**Expected behavior**

Picture is both clear and concise, hopefully
[https://imgur.com/0L8Mu2h](url)

**Info (please complete the following information):**
 - Modpack Version: 2.77
 - Server or Singleplayer: Server
 - Version the world was created on: 2.77

**Additional context**
Add any other context about the problem here and your thoughts about what could have caused it.
I am not a shady guy, I am not making illegal furnaces I swear

**Comment by unicornbloods** [comment]
"Fatally missing registry entries" 
also occurs

**Comment by LemADEC** [comment]
Issue reproduced with multiple client after updating server from 1.76.2 to 1.77.0.
```
[11:16:23] [Client thread/ERROR]: Unidentified mapping from registry livingenchantment:personalityregistry
[11:16:23] [Client thread/ERROR]:     livingenchantment:evil: 5
```
Configuration for the livingenchantment mod is the same between client and server.

**Comment by LemADEC** [comment]
The evil personality needs to be removed manually when updating the server configuration.

**Comment by unicornbloods** [comment]
I am confused. This doesn't seem relevant to my issue at all xD

**Comment by LemADEC** [comment]
the initial error is "Fatally missing registry entries". If you try to join again, since the registry is corrupted from the previous error, you get a furnace inception error.

**Comment by unicornbloods** [comment]
I see, so I remove The evil personality from the server? Even though it is a server I just started on 2.77?

**Comment by unicornbloods** [comment]
There isn't an evil personality on server or client

**Comment by unicornbloods** [comment]
~~~Fixed with 2.77.1~~~
